examples/tests/valid/if.w

```diff
@@ -0,0 +1,18 @@
+bring cloud;
+
+let x = 1;
+let var preflight_ok = false;
+if x == 1 {
+  print("x is 1");
+  preflight_ok = true;
+}
+assert(preflight_ok);
+
+new cloud.Function(inflight (s: str): str => {
+  let var inflight_ok = false;
+  if x == 1 {
+    print("x is 1");
+    inflight_ok = true;
+  }
+  assert(inflight_ok);
```

Comment:
This **inflight** assert never really runs. To test this we need some deployment trigger to run the cloud function and we need to deploy it to sim and verify somehow the cloud function passed the assertion. I think we need a new issue for this or am I missing something? @eladb? @staycoolcall911?

Reply:
I comment the line `inflight_ok = true;` and test fail

```bash

        FAIL  src/cli.test.ts > wing test if.w (--target sim)
       Error: promise rejected "fail | if.w | root/test
       Error: assertion failed: 'inflight_ok'
           at evalmachine.<anonymous>:32:23
           at Handler.handle (evalmachine.<anonymous>:33:15)
           at exports.handler (evalmachine.<anonymous>:41:6)
           at evalmachine.<anonymous>:44:9
           at Script.runInContext (node:vm:141:12)
           at Object.runInContext (node:vm:297:6)
           at Object.activity (/home/marcio/code/wing/tools/hangar/tmp/node_modules/@winglang/sdk/lib/target-sim/function.inflight.js:85:27)
           at Object.withTrace (/home/marcio/code/wing/tools/hangar/tmp/node_modules/@winglang/sdk/lib/testing/simulator.js:107:50)
           at Function.invoke (/home/marcio/code/wing/tools/hangar/tmp/node_modules/@winglang/sdk/lib/target-sim/function.inflight.js:82:29)
           at Simulator.runTest (/home/marcio/code/wing/tools/hangar/tmp/node_modules/@winglang/sdk/lib/testing/simulator.js:295:22)
       " instead of resolving
```

Reply:
But I find a `statemetns_if.w` test file 

I will put my tests in this file and remove the others, making all `if` test in one file

Reply:
I see we invoke `cloud.Function`s in our test harness. Wasn't aware of this. Cool!